airflow/www_rbac/views.py

```diff
@@ -147,6 +147,33 @@ def render(self, template, **context):
 
 
 class Airflow(AirflowBaseView):
+    @expose('/health')
+    @provide_session
+    def health(self, session=None):
+        from sqlalchemy import func
+        DM = models.DagModel
+        payload = {}
+
+        latest_scheduler_run = None
+        payload['metadatabase'] = {'status': 'healthy'}
+        try:
+            latest_scheduler_run = session.query(func.max(DM.last_scheduler_run)).scalar()
+        except Exception as _:
+            payload['metadatabase']['status'] = 'unhealthy'
+
+        if not latest_scheduler_run:
+            scheduler_status = 'unhealthy'
+        else:
+            if timezone.utcnow() - latest_scheduler_run <= timedelta(seconds=30):
```

Comment:
same comment on why we pick 30 seconds as magic number(needs some comments). And it would be good if we put it in a config value so that it allows users to change.